autopush/endpoint.py

```diff
@@ -254,6 +278,43 @@ def _db_error_handling(self, d):
         d.addErrback(self._response_err)
         return d
 
+    def _store_auth(self, jwt, result):
+        if jwt.get('exp') and jwt['exp'] < time.time():
+            raise VapidAuthException("Invalid bearer token: Auth expired")
+        self._client_info.update(jwt)
+        return result
+
+    def _invalid_auth(self, fail):
+        if isinstance(fail.value, VapidAuthException):
+            raise fail.value
+        message = fail.value.message or repr(fail.value)
+        if isinstance(fail.value, AssertionError):
+            message = "A decryption error occurred"
+        log.msg("Invalid bearer token: " + message, **self._client_info)
+        raise VapidAuthException("Invalid bearer token: " + message)
+
+    def _process_auth(self, result):
```

Comment:
Did all my comments about how to refactor this function disappear from the client_info commit? I think they'd really clean this function up a bit more.